index.html

```diff
@@ -463,6 +510,10 @@ <h2>
         is only [=exposed=] in [=secure contexts=] (such as `https://`
         schemes).
         </li>
+        <li>Third-party contexts require that they be [=allowed to use=] the
+        {{Navigator/share()}} method is one of the mechanisms described in
+        [[[permissions-policy]]].
```

Comment:
This also looks new, maybe better to split out.

Reply:
https://github.com/w3c/web-share/pull/210